docker/run.sh

```diff
@@ -1,12 +1,18 @@
 #!/bin/sh
+#
+# Run the docker container.
+
 . docker/env.sh
 docker stop $CONTAINER_NAME
 docker run \
   -dit \
   --gpus all \
   -v $PWD:/workspace \
-  -p 6008:6008 \
+  -p $HOST_PORT:$CONTAINER_PORT \
   --name $CONTAINER_NAME\
   --rm \
   --shm-size=2g \
   $IMAGE_NAME
+docker exec \
+  -dit \
```

Comment:
Is this line necessary?

Reply:
I think it's necessary. If the implementation is as follows, the run process will stop and the container will be removed if an error occurs in init.sh. Therefore, I decided to separate the processes for safety.

```sh
#!/bin/sh
#
# Run the docker container.

. docker/env.sh
docker stop $CONTAINER_NAME
docker run \
  -dit \
  --gpus all \
  -v $PWD:/workspace \
  -p $MLFLOW_HOST_PORT:$MLFLOW_CONTAINER_PORT \
  --name $CONTAINER_NAME\
  --rm \
  --shm-size=2g \
  $IMAGE_NAME sh /workspace/docker/init.sh
```

Reply:
I mean `-dit` is necessary when running `docker exec`?

Reply:
Without this option, the result of init.sh execution will be printed to the terminal when the container starts to run. I think this is inconvenient because the users need to open another tab in the terminal to run exec.sh.

Reply:
That's right, but usually only `-d` option is used to start web application or web server.  Is there any reason to use `-it`?

Reply:
I see. I misunderstood that "d" is always used with "it". I'll fix it.